linux_os/guide/services/mail/postfix_client/postfix_client_configure_mail_alias/rule.yml

```diff
@@ -18,9 +18,11 @@ identifiers:
     cce@rhel6: 80508-5
 
 references:
-    disa@rhel6: "366"
+    disa: "366"
     srg@rhel6: SRG-OS-999999
     stigid@rhel6: RHEL-06-000521
+    nist: CM-6(b)
```

Comment:
CM-6(b) is an organization control, stating orgs must implement the controls created in CM-6(a). Individual configuration checks should not map to CM-6(b), this mapping should be removed.

ref: https://nvd.nist.gov/800-53/Rev4/control/CM-6